terraform/cloudfoundry/security-groups.tf

```diff
@@ -94,7 +94,6 @@ resource "aws_security_group" "web" {
 
     cidr_blocks = [
       "${compact(var.admin_cidrs)}",
-      "${compact(var.tenant_cidrs)}",
```

Comment:
Could you clarify, Is this one not getting replaced with the new `api_access_cidrs ` like the other two? Why?